kmultiversx/src/kmultiversx/kdist/mx-semantics/data/list-bytes.k

```diff
@@ -47,6 +47,12 @@ module LIST-BYTES-EXTENSIONS
 
   syntax WrappedBytes ::= ListBytes "[" Int "]" "orDefault" WrappedBytes
           [ function, total, symbol(ListBytes:getOrDefault) ]
+// ---------------------------------------------------------------------
+  rule L:ListBytes[I:Int] orDefault _:WrappedBytes => L[I]
+    requires 0 -Int size(L) <=Int I andBool I <Int size(L)
```

Comment:
I would add `[concrete]` unless you need it to also work on symbolic inputs.

Reply:
> I would add `[concrete]` unless you need it to also work on symbolic inputs.

I think I want it to work on symbolic inputs, too. At least the list `L` might not be fully-concrete but has a determined element at index `I`.
This is not a simplification but a defining equation. I don't think we can add the `[concrete]` attribute without making it a simplification (and then we would need another set of defining equations).